packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.Props.ts

```diff
@@ -59,4 +60,155 @@ export interface IComboBoxProps extends ISelectableDroppableTextProps<IComboBox>
    * The IconProps to use for the button aspect of the combobox
    */
   buttonIconProps?: IIconProps;
+
+  /**
+   * Theme provided by HOC.
+   */
+  theme?: ITheme;
+
+  /**
+   * Custom styles for this component
+   */
+  styles?: IComboBoxStyles;

```

Comment:
It is a good idea. Just saying, I have written styles in the interface, that I have not written default styles for - to allow for a user to be able to customize it. 
I could, technically, just put in an empty object for each, to avoid the compiler error, but then the compiler error would not be of too much use too. 
Let me know your thoughts.